lib/kaguya/core.ex

```diff
@@ -128,6 +133,7 @@ defmodule Kaguya.Core do
     cancel_server_timer(state)
     socket = reconnect()
     send(self(), :init)
+    send(self(), :rejoin_chan)
```

Comment:
There might a question whether we should join only after sending nick?
Since on `:init` we start task to send nick and etc, maybe we should rejoin only after that?